src/backend/marsha/bbb/utils/bbb_utils.py

```diff
@@ -304,12 +304,15 @@ def get_meeting_infos(classroom: Classroom):
             else:
                 api_response["attendees"] = [attendees]
 
-        classroom.save(update_fields=["started"])
+        classroom.infos = api_response
```

Comment:
I think you can add an assert to test_get_meeting_infos.py to test if the the property is rightfully set